src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/Feedback.tsx

```diff
@@ -29,8 +29,9 @@ export const Feedback = ({ activeTab, setActiveTab, onClose, selectedEntity }: F
   });
 
   const tabs = useMemo(
-    () => getFeedbackTabs(selectedEntity, assessment),
-    [selectedEntity, assessment],
+    () => getFeedbackTabs({ selectedEntity, assessment }),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [selectedEntity.id, selectedEntity.isFlow, assessment],
```

Comment:
I think we can simplify this by using just `selectedEntity` instead of `selectedEntity.id` and `selectedEntity.isFlow`. Since `selectedEntity` only has these two properties and no others, there's no need to destructure it further for this calculation.